cost_calculator/instance-data/aws_instance_data.json

```diff
@@ -7296,6 +7296,16 @@
             "gpu": 0,
             "cpu": 8
         },
+        {
+            "baseline": 8,
+            "generation": "current",
+            "price": 0.476,
+            "memory": 32.0,
+            "instanceType": "m5n.2xlarge",
+            "burstable": false,
+            "gpu": 0,
+            "cpu": 8
+        },
```

Comment:
Instance data is generated here: https://github.com/elotl/kip/tree/master/scripts/create_instance_data

If an instance type is missing, we should fix it there.

Reply:
Thanks for pointing this out! Generated missing data with scripts from there. Do we want to update files in kip as well?

Reply:
Let's treat that separately, since reviewing changes in instance data is pretty manual.